pkg/network/common/egress_dns.go

```diff
@@ -5,89 +5,110 @@ import (
 	"sync"
 	"time"
 
+	networkv1 "github.com/openshift/api/network/v1"
 	ktypes "k8s.io/apimachinery/pkg/types"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
-
-	networkv1 "github.com/openshift/api/network/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/klog"
 )
 
-type EgressDNSUpdate struct {
+type PolicyUpdate struct {
```

Comment:
Keep the old name, please; it has to be unique across `pkg/network/common/`, so `EgressDNSUpdate` is better. (Likewise with `PolicyUpdates`, although you don't actually need to have a type alias there, you can just use "`chan []EgressDNSUpdate`", etc

Reply:
Will change the name back to `EgressDNSUpdate`, I prefer the type alias for brevity though.